### ISSUE TITLE
Content by search webpart on a team site page is not getting transformed by Site collection modernization script

### ISSUE THREAD

**Issue by abhishektyagi9** [SharePoint/sp-dev-modernization#180]
#### Category

[ ] Bug
[ ] Enhancement

#### Problem Area

[ ] Page Transformation: Error during the use of page transformation from PnP PowerShell

Content by search webpart is not getting transformed to highlighted content web part using page transformation script.

#### Expected or Desired Behavior

Content search webpart should be replaced by highlighted content web part

#### Steps to Reproduce

I ran following script which is running successfully but Content by Search webpart is not getting transformed to Highlighted content webpart.

https://github.com/SharePoint/sp-dev-modernization/blob/master/Scripts/ModernizeSiteCollection/modernizesitecollection.ps1

**Comment by jansenbe** [comment]
Hi @abhishektyagi9 ,

Are you doing a cross-site modernization (create pages in another site collection) or an in-place modernization (create pages in the same site collection)? For cross-site modernization it's expected that the content by search web part is not transformed as mentioned here: https://docs.microsoft.com/en-us/sharepoint/dev/transform/modernize-userinterface-site-pages-webparts.

In case you're using in-place modernization can you then check that you're using the latest version and see if you still have the issue?

**Comment by jansenbe** [comment]
Hey @abhishektyagi9 ,

Wanted to verify if the above answer got you unblocked?

**Comment by jansenbe** [comment]
Hi @abhishektyagi9 ,

I'm closing this issue due to no further feedback. If you're still stuck then please create a new issue or re-open this one.

**Comment by abhishektyagi9** [comment]
Yes thanks it was fixed.I was doing in place only but seems like powershell code installs old versions of SharePoint pnp but when I updated issue was fixed.
Thanks for follow-up.
